src/components/CardList.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
+import { getFolderData } from "../Api/Api";
+
+function Card() {
+  return;
+}
+
+export function CardList() {
+  const [folderData, setFolderData] = useState(null);
+  useEffect(getFolderData(setFolderData), []);
```

Comment:
App.js에서 해주신 것처럼, useEffect 훅의 첫번째 인자에는 함수를 전달해야 합니다.  
작성하신 코드는 getFolderData 함수의 호출 결과를 전달하고 있네요
```suggestion
  useEffect(() => {
    getFolderData(setFolderData);
  }, []);
```